### PR TITLE
Change rules for LOGS and changing the assessor

### PR DESCRIPTION
**Pull request by lloback**
This PR changes permissions for two tasks that I was working on:

LOGS
Change ASSESSORS

### CODE REVIEW

File: database/firestore.rules [hunk 1]
```diff
@@ -121,10 +121,15 @@ service cloud.firestore {
       allow write: if userIsAuthenticated() && userIsJAC();
     }
 
+    match /logs/{id}/{document=**} {
+      // allow candidates update their own logs
+      allow get, read, create, write, update: if userIsAuthenticated();
```
Comment (warrensearle) [inline database/firestore.rules:126]: There are un-necessary verbs here:
`read` and `write` are high-level and cover the relevant lower level verbs
`read` implies `list` and `get`
`write` implies `create`, `update`, `delete`

File: database/firestore.rules [hunk 2]
```diff
@@ -121,10 +121,15 @@ service cloud.firestore {
       allow write: if userIsAuthenticated() && userIsJAC();
     }
 
+    match /logs/{id}/{document=**} {
+      // allow candidates update their own logs
```
Comment (warrensearle) [inline database/firestore.rules:125]: This doesn't restrict candidate to only updating their own log

**Review by warrensearle — CHANGES_REQUESTED** [review]

**Comment by lloback** [comment]
The only thing that seems to works here is:

```
match /logs/{document=**} {
      // allow candidates update their own logs
      allow read, write: if userIsAuthenticated();
      allow read: if userIsAuthenticated() && userIsJAC();
    }
```

**Comment by lloback** [comment]
I've tried different combinatiosn and it doesn't work:

```
match /logs/login/{id} {
      allow read, write: if userIsAuthenticated() && currentUser() == id; // DON"T WORK
      allow read: if userIsAuthenticated() && userIsJAC();
    }
```

```
match /logs/{document=**}/{id} {
      allow read, write: if userIsAuthenticated() && currentUser() == id; // DON"T WORK
      allow read: if userIsAuthenticated() && userIsJAC();
    }
```

**Comment by warrensearle** [comment]
I think the following should do the trick:
```
match /logs/login/{id}/{logId} {
      // allow candidates update their own logs
      allow read, write: if userIsAuthenticated() && currentUser() == id;
      allow read: if userIsAuthenticated() && userIsJAC();
    }
```

File: database/firestore.rules (hunk 1 above)
Comment (lloback) [reply database/firestore.rules]: this is implemented.

**Review by warrensearle — APPROVED** [review]
👍

**Pull request merged**